### PR TITLE
chore(snapshot): file generation to use path join function

### PR DESCRIPTION
**Pull request by valentin-mladenov**
This makes snapshot generation windows OS friendly.

### CODE REVIEW

File: scripts/write-story-list.ts [hunk 1]
```diff
@@ -52,7 +53,8 @@ async function main() {
   });
 
   // And write a file for the storybook-visual-regression-test `playwright` test to read.
-  fs.writeFileSync('./dist/docs/stories.json', JSON.stringify(storyIds, undefined, 2));
+  const storiesFileString = path.join('.', 'dist', 'docs', 'stories.json');
```
Comment (kevinbuhmann) [inline scripts/write-story-list.ts:56]:
```suggestion
  const storiesFilePath = path.join('.', 'dist', 'docs', 'stories.json');
```

Please rename in both files.

Comment (valentin-mladenov) [reply scripts/write-story-list.ts]: fixed

**Review by kevinbuhmann — APPROVED** [review]

**Review by kevinbuhmann — CHANGES_REQUESTED** [review]
Wait, shouldn't the same change be made for the last line of the `visual-snapshots.spec.ts` file?

```ts
fs.writeFileSync(`./tests/snapshots/used-screenshot-paths-${matrixKey}.txt`, usedScreenshotPaths.join('\n'));
```

File: tests/visual-snapshots.spec.ts [hunk 2]
```diff
@@ -60,4 +61,7 @@ const takeFullPageScreenshot = (component, storyName) => {
   return ScreenshotOptions[component]?.fullPageScreenshot || ScreenshotOptions[storyName]?.fullPageScreenshot;
 };
 
-fs.writeFileSync(`./tests/snapshots/used-screenshot-paths-${matrixKey}.txt`, usedScreenshotPaths.join('\n'));
+fs.writeFileSync(
+  path.join(`.`, `tests`, `snapshots`, `used-screenshot-paths-${matrixKey}.txt`),
+  usedScreenshotPaths.join('\n')
+);
```
Comment (kevinbuhmann) [inline tests/visual-snapshots.spec.ts:67]:
```suggestion
constunusedScreenshotsFilePath = path.join(`.`, `tests`, `snapshots`, `used-screenshot-paths-${matrixKey}.txt`);
fs.writeFileSync(unusedScreenshotsFilePath,usedScreenshotPaths.join('\n'));
```

extract this one to a variable like the stories file path?

Comment (valentin-mladenov) [reply tests/visual-snapshots.spec.ts]: corrected

**Review by kevinbuhmann — APPROVED** [review]

**Pull request merged**